### ISSUE TITLE
Accessibility - Bookmark Page - Needs to Remove Tabindex="-1" for Bookmark Links

### ISSUE THREAD

**Issue by jfc3-dol** [USDepartmentofLabor/Comply-Chain#206]
While I tested issue #204 I did noticed in the web version I can not now tab to any of the bookmarked items.

People can not tab to the bookmarked links which have a tabindex="-1" and should be removed. Not sure why these are there because I don't remember them from before.

Please see the attached screenshot.

<img width="960" alt="ialb - comply chain - bookmark page - need to remove tabindex equal negative one from bookmark links" src="https://user-images.githubusercontent.com/10423487/128889846-761e8e38-84ad-4eef-8627-f65d64407a6e.png">

**Comment by jfc3-dol** [comment]
@pbhatt17 

There is no need for a tabindex on the bookmark links because links receive focus when a person tabs to them on their own. I can tab to the links now as expected.

Can you please remove the tabindex="0"?

Keeping the issue open until the tabindex is removed.

**Comment by jfc3-dol** [comment]
@pbhatt17 

The tabindex="0" has been removed from the link as requested, so I'm going to close the issue.
